### DISCUSSION TITLE
오픈채팅방 검색 기능 관련

### DISCUSSION THREAD

**Discussion by MoonJR** [storycraft/node-kakao#310] [Q&A] [answered]
안녕하세요. 

카카오톡을 보던 중 기능중에 오픈 채팅방을 검색하는 기능이 있어서 혹시 node-kakao v4 에 해당 기능도 포함하고 있는지 굼금하여 문의드립니다.

문서를 살펴보았지만 이렇다할 메소드를 찾지 못하여 문의 드립니다~!

**Comment by raravel** [comment, marked as answer]
V3 에 있는 API를 사용해서 주소를 직접 만들 순 있습니다.

```
https://open.kakao.com/c/search/unified?q=[검색문자열]&s=l&p=[페이지]&c=[페이지당 보일 검색 결과 수]&expectLock=[잠금방 여부 Y or N]
```

**Reply by MoonJR** [reply]
@raravel 잘 동작하네요! 감사합니다!
